lighthouse-core/lib/network-request.js

```diff
@@ -261,6 +263,7 @@ class NetworkRequest {
   onRedirectResponse(data) {
     if (!data.redirectResponse) throw new Error('Missing redirectResponse data');
     this._onResponse(data.redirectResponse, data.timestamp, data.type);
+    this.redirectedResourceType = this.resourceType;
```

Comment:
can we get this information from observing the `.redirectDestination.resourceType` or is it different?

Reply:
Hmm, I tried something similar but it wasn't populated when running simulation tests. Let me double check

Reply:
That seems to work! Thanks for the suggestion